WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -159,13 +182,18 @@ private extension ItemListView {
 private extension ItemListView {
     enum Constants {
         static let titleFont: Font = .system(size: 40, weight: .bold, design: .default)
-        static let bannerTitleFont: Font = .system(size: 26, weight: .bold, design: .default)
-        static let bannerHeight: CGFloat = 120
-        static let infoIconSize: CGFloat = 48
+        static let bannerTitleFont: Font = .system(size: 24, weight: .bold, design: .default)
```

Comment:
My observation with both PRs is that designs use some different fonts than the default Apple ones. Not sure if you observe the same. Of course, if needed we could make some replacements later.